### PR TITLE
Post: new flyout in beta

### PR DESCRIPTION
**Pull request by humitos**
Ideally, this post should have been published yesterday together with the deploy -- but I haven't had the time to write it down.

I'm happy to publish it tomorrow.

I think we can write a small blog post like this one each time we enable a new addons by default.

📚  Rendered version: https://readthedocs-blog--240.org.readthedocs.build/flyout-beta/

### CODE REVIEW

File: flyout-beta.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Analytics
+- Non latest version warning banner
+- PR warning banner
+- Search as you type
+- Sponsorship via EthicalAds
+- Visual documentation diff
+
+
+As this work is still in beta, we haven't rolled it out to all the projects yet.
+We have decided to start with projects using ``build.commands`` config key (`read more <https://docs.readthedocs.io/en/latest/build-customization.html>`_)
```
Comment (agjohnson) [inline flyout-beta.rst:59]: This is an important aspect that feels a little buried. I think a small restructure of the flow of this post might help, while still allowing the post to get into technical details. That is, introduce the problem and solution early in the post, and then get into technical details.

**Review by agjohnson — COMMENTED** [review]
I'll come back to some review, but just went through for high level notes for now. I'll come back for suggestions.

I think the content in the post is good, and a slight restructure probably solves most of my notes really. This post is leaning more technical, which is still a good focus, but it might more helpful to our users if we introduce the problem and solution right up front.

That is, structuring:

- We're doing a beta
- What problem we're solving
- How you can use the new addons
- Why we are changing our implementation -- the technical details already here

**Comment by agjohnson** [comment]
I pushed up my changes to restructure and will publish next.

**Review by agjohnson — APPROVED** [review]
Approving to get this out today.

File: flyout-beta.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   :category: Feature announcement
+
+New flyout re-desing in beta
+============================
+
+We are happy to announce the **beta release of a completely new flyout** that was re-written and re-designed from scratch.
+
+After lot of hard work, we are happy to start rolling out the work of the new flyout to some projects.
+Judging by its style, there are not big differences since it looks pretty close to the current one.
+The main visual difference is that it now shows a Read the Docs logo when it's collapsed and expanded.
```
[the remark this answers is not in the document]
Comment (humitos) [reply flyout-beta.rst]: Hrm, interesting. I re-read this paragraph and I don't find it negative or critical. By that, do you mean that saying there are "no visual differences" is not good?

File: flyout-beta.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+JS file is injected at serve time
+    Previously, we were injecting CSS/JS files at build time by using a custom Sphinx extension (``readthedocs-sphinx-ext``) that we installed on behalf of the user when building on Read the Docs.
+    On the other hand, for MkDocs projects, we were manipulating the ``mkdocs.yml`` file to add these files.
+
+    Now, we are injecting *only one* self-contained JS file called ``readthedocs-addons.js`` at serving time via Cloudflare.
+    This allows us to support any documentation tool (e.g. Docusaurus, Docsify, etc) without having to create a specific integration with it,
+    helping with our goal of making our platform documentation tool agnostic.
+
+The flyout is fully rendered based on a JSON API call
```
[the remark this answers is not in the document]
Comment (humitos) [reply flyout-beta.rst]: Yes. I wanted to mention and make emphasis on this but then I thought that we are not allowing this yet 😅, and I didn't know exactly how to write it

File: flyout-beta.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (agjohnson) [reply flyout-beta.rst]: Aye, I guess "negative" is not the right word here. I read the first sentence as downplaying the changes at very least though, so maybe just not as positive as we could be.

[the remark this answers is not in the document]
Comment (humitos) [reply flyout-beta.rst]: Gotcha! Thanks 👍

**Pull request merged**